### PR TITLE
removed every mention of angstrom, fixes issue #14

### PR DESCRIPTION
**Pull request by xzackli**
As described in #14, Unitful.jl now has an angstrom unit, so we should remove it from this package to avoid annoying warnings when using the two together. As you can see from the diffs, I just typed `rg -i angstrom` and removed every line with a mention of it.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
 *2017-09-27*
 
 * Initial tag.
-* List of units: erg, dyn, arcmin, arcsec, Julian year, AU, light year, parsec, Jansky, Angstrom,
+* List of units: erg, dyn, arcmin, arcsec, Julian year, AU, light year, parsec, Jansky,
```
Comment (giordano) [inline NEWS.md]: This was good, it's part of the documentation :wink:

File: test/runtests.jl [hunk 2]
```diff
@@ -13,7 +13,6 @@ const u = UnitfulAstro
         @test 365.25*u.d == 1*u.yr
         @test 1*u.ly == 1*u.c*u.yr
         @test 1*u.pc == 1*u.AU/u.arcsecond
-        @test 1*u.angstrom == 1e-10*u.m
```
Comment (giordano) [inline test/runtests.jl:16]: The unit is no more defined by this package, nevertheless I think we may want to be sure it's available, as it's important for astronomical applications.

**Review by giordano — CHANGES_REQUESTED** [review]
Thank you so much!  In addition to the comments below, you need to bump the required version of Unitful to 0.18

File: NEWS.md (hunk 1 above)
Comment (xzackli) [reply NEWS.md]: Wow, that's embarrassing! You can tell that I really did just search for every mention of angstrom and remove it, hehe.

**Comment by xzackli** [comment]
I've committed the requested changes, sorry it took a bit (end of semester, had to give a final and grade it).

**Comment by xzackli** [comment]
The angstrom test fails for the appveyor 32-bit Windows build. Could it be that the rational number type (which I assume is made of two ints) defining the angstrom in [Unitful](https://github.com/PainterQubits/Unitful.jl/blob/master/src/pkgdefaults.jl#L185) in relation to the meter involves a factor of 10^10, which exceeds 2^31?

File: Project.toml [hunk 3]
```diff
@@ -8,7 +8,7 @@ Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 UnitfulAngles = "6fb2a4bd-7999-5318-a3b2-8ad61056cd98"
 
 [compat]
-Unitful = "≥ 0.17.0"
+Unitful = "≥ 0.18.0"
```
Comment (giordano) [inline Project.toml]:
```suggestion
Unitful = "0.18.0"
```

File: Project.toml [hunk 4]
```diff
@@ -8,7 +8,7 @@ Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
...
 julia = "≥ 1.0.0"
```
Comment (giordano) [inline Project.toml]:
```suggestion
julia = "1.0.0"
```

**Review by giordano — APPROVED** [review]

**Comment by giordano** [comment]
Thank you so much and sorry for the late review!

**Pull request merged**